src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -219,22 +219,44 @@ object PlayReleaseBase extends AutoPlugin {
         }
       }
 
-      Seq[ReleaseStep](
-        checkSnapshotDependencies,
-        inquireVersions,
-        runTest,
-        releaseStepTask(playBuildExtraTests in thisProjectRef.value),
-        setReleaseVersion,
-        commitReleaseVersion,
-        tagRelease,
-        publishArtifacts,
-        releaseStepTask(playBuildExtraPublish in thisProjectRef.value),
-        ifDefinedAndTrue(playBuildPromoteBintray, releaseStepTask(bintrayRelease in thisProjectRef.value)),
-        ifDefinedAndTrue(playBuildPromoteSonatype, releaseStepCommand("sonatypeRelease")),
-        setNextVersion,
-        commitNextVersion,
-        pushChanges
-      )
+      if (sbtPlugin.value) {
+        Seq[ReleaseStep](
+          checkSnapshotDependencies,
+          inquireVersions,
+          runClean,
+          releaseStepCommandAndRemaining("^ test"),
+          releaseStepCommandAndRemaining("^ scripted"),
+          setReleaseVersion,
+          commitReleaseVersion,
+          tagRelease,
+          releaseStepCommandAndRemaining("^ publishSigned"),
+          releaseStepTask(playBuildExtraPublish in thisProjectRef.value),
+          ifDefinedAndTrue(playBuildPromoteBintray, releaseStepTask(bintrayRelease in thisProjectRef.value)),
+          ifDefinedAndTrue(playBuildPromoteSonatype, releaseStepCommand("sonatypeRelease")),
+          setNextVersion,
+          commitNextVersion,
+          pushChanges
+        )
+      } else {
+        Seq[ReleaseStep](
```

Comment:
My only comment is that this could be refactored a bit to show that there's common code in both branches of the condition.

E.g.

```scala
Seq[ReleaseStep](
  checkSnapshotDependencies,
  ...
  runClean
) ++ if (sbtPlugin.value) {
  // Plugins need these steps because ... ???
  Seq[ReleaseStep](
    releaseStepCommandAndRemaining("^ test"),
    ...
    releaseStepCommandAndRemaining("^ publishSigned")
  )
} else {
  // Normal artifacts need these steps because ... ???
  Seq[ReleseStep](
    runTest,
    ...
    publishArtifacts
  )
} ++ Seq[ReleaseStep](
  releaseStepTask(playBuildExtraPublish in thisProjectRef.value),
  ...
  pushChanges
)